### PR TITLE
Ansible: Add second manage & other improvements

### PR DESCRIPTION
**Pull request by quartje**
A second manage instance is configured in the SPdashboard configuration.
Some minor Ansible fixes are added to this PR

### CODE REVIEW

File: ansible/roles/spdashboard/defaults/main.yml [hunk 1]
```diff
@@ -14,4 +14,6 @@ spdashboard_saml_metadata_privatekey: "%kernel.root_dir%/../vendor/surfnet/stepu
 spdashboard_support_email: coin-logs-dev@list.surfnet.nl
 
 # In order to be able to use Webpack Encore 0.21, Node > 6 is required.
-spdashboard_node_version: v11.1.0
\ No newline at end of file
+spdashboard_node_version: v11.1.0
```
Comment (MKodde) [inline ansible/roles/spdashboard/defaults/main.yml]: This version pin can be removed now. Maybe replace it with the RPM location for the Node JS version you are installing manually:

'https://rpm.nodesource.com/pub_11.x/el/7/x86_64/nodesource-release-el7-1.noarch.rpm'

Comment (quartje) [reply ansible/roles/spdashboard/defaults/main.yml]: That's a good idea!

**Comment by quartje** [comment]
Currently we're the only ones using nodejs for building apps. No need to take other into consideration :)

**Comment by MKodde** [comment]
Merging!

**Pull request merged**